packages/client/types/errors.ts

```diff
@@ -0,0 +1,6 @@
+export const SprintPokerErrors = {
+  fixMissingFieldError:
+    'Couldn’t fix the missing field! In Jira, use "Find my field" to determine the error',
+  updateFieldError: (fieldName: string) =>
+    `Update failed! In Jira, add the field "${fieldName}" to the Issue screen.`
+}
```

Comment:
I don't particularly like this solution, but I can't think of anything better 🤔

Reply:
ahhh good catch!

because jira's "context" could also be a cause, I think the following error message is no longer true: `Update failed! In Jira, add the field "${fieldName}" to the Issue screen.` because it could exist on the issue screen & the context could be wrong.

since both error messages have the same root cause (misconfigured jira) and same solution (goto find my field & see the error), let's use a single error message for them both.

Maybe something like: `Jira rejected the field update. In Jira, use "Find my field" to determine the cause`

what do you think?

Reply:
Sounds great! I'll update the places with a single error message and it'll be 👌